api/MouseEvent.json

```
@@ -237,10 +237,10 @@
               "version_added": "11.3"
             },
             "samsunginternet_android": {
-              "version_added": false
+              "version_added": "5.0"
```

Is this based on the mirroring script? If so can you file an issue, because based on manual mirroring it should be:

```suggestion
              "version_added": "4.0"
```